ItemGenerator/Models/RarityProperites.cs

```diff
@@ -0,0 +1,11 @@
+using ExampleItemGenerator.Models;
+
+namespace ExampleItemGenerator.Services {
+    public struct RarityProperties {
```

Comment:
Nie jestem pewien czy `struct` to najlepsze rozwiązanie w tym miejscu – używa się ich naprawdę rzadko w typowym kodzie biznesowym, a prawdopodobnie najlepszym rozwiązaniem dla czegoś tego typu jak tutaj byłby `record`, których w tym momencie jeszcze nie omawialiśmy.